packages/docs/content/docs/tutorials/supabase-mcp-server.mdx

```diff
@@ -0,0 +1,584 @@
+---
+title: 'Building a Supabase MCP Server for Natural Language SQL Queries in Pochi'
+---
+
+# Building a Supabase MCP Server for Natural Language SQL Queries in Pochi
+
+Often, when working with teams to debug code or build features, you need to look up data from your database. This may include checking if a recent migration seeded correctly, inspecting a particular table schema, or verifying specific column values. Normally, this means breaking your flow, opening the SQL client, writing queries by hand, or involving a co-worker.
+
+This tutorial is designed to help you avoid that. We’ll build a system where you can prompt Pochi right inside your code editor to answer all your database related queries using natural language prompts. For example: “Give me the table schema of the ‘sales_report’ table.”
```

Comment:
The introduction could be refined to better frame the tutorial's goal. Consider rephrasing this to emphasize that the main purpose is to showcase how to build a Pochi MCP server from scratch, using Supabase as a practical example. This would clarify that it's an educational piece about Pochi's extensibility, not an attempt to create an official or replacement Supabase integration.